examples/xxz/main.jl

```diff
@@ -0,0 +1,99 @@
+using Markdown #hide
+md"""
+# Néel order in the $U(1)$-symmetric XXZ model
+
+Here, we want to look at a special case of the Heisenberg model, where the $x$ and $y$
+couplings are equal, called the XXZ model
+
+```math
+H_0 = J \big(\sum_{\langle i, j \rangle} S_i^x S_j^x + S_i^y S_j^y + \Delta S_i^z S_j^z \big) .
+```
+
+For appropriate $\Delta$, the model enters an antiferromagnetic phase (Néel order) which we
+will force by adding staggered magnetic charges to ``H_0``. Furthermore, since the XXZ
+Hamiltonian obeys a $U(1)$ symmetry, we will make use of that and work with $U(1)$-symmetric
+PEPS and CTMRG environments. For simplicity, we will consider spin-$1/2$ operators.
+
+But first, let's make this example deterministic and import the required packages:
+"""
+
+using Random
+using TensorKit, PEPSKit
+using MPSKit: add_physical_charge
+Random.seed!(2928528935);
+
+md"""
+## Constructing the model
+
+Let us define the $U(1)$-symmetric XXZ Hamiltonian on a $2 \times 2$ unit cell with the
+parameters:
+"""
+
+J = 1.0
+Delta = 1.0
+spin = 1//2
+symmetry = U1Irrep
+lattice = InfiniteSquare(2, 2)
+H₀ = heisenberg_XXZ(ComplexF64, symmetry, lattice; J, Delta, spin);
+
+md"""
+This ensures that our PEPS ansatz can support the bipartite Néel order. As discussed above,
+we encode the Néel order directly in the ansatz by adding staggered auxiliary physical
+charges:
+"""
+
+S_aux = [
+    U1Irrep(-1//2) U1Irrep(1//2)
+    U1Irrep(1//2) U1Irrep(-1//2)
+]
+H = add_physical_charge(H₀, S_aux);
+
+md"""
+## Specifying the symmetric virtual spaces
+
+Before we create an initial PEPS and CTM environment, we need to think about which
+symmetric spaces we need to construct. Since we want to exploit the global $U(1)$ symmetry
+of the model, we will use TensorKit's `U1Space`s where we specify dimensions for each
+symmetry sector. From the virtual spaces, we will need to construct a unit cell (a matrix)
+of spaces which will be supplied to the PEPS constructor. The same is true for the physical
+spaces, which can be extracted directly from the Hamiltonian `LocalOperator`:
+"""
+
+V_peps = U1Space(0 => 2, 1 => 1, -1 => 1)
+V_env = U1Space(0 => 6, 1 => 4, -1 => 4, 2 => 2, -2 => 2)
+virtual_spaces = fill(V_peps, size(lattice)...)
+physical_spaces = H.lattice
```

Comment:
```suggestion
physical_spaces = physicalspace(H)
```
This might need to be defined, (imported from MPSKit)

Reply:
I now defined `physicalspace(::LocalOperator)` and used it consistently throughout the code.

Reply:
I imported that now from MPSKit and it does work. I hope that's what you mean?